quay-enterprise/custom-cert-kube-apps.md

```diff
@@ -0,0 +1,40 @@
+## Using Helm Registry Plug-in with a self-signed certificate
+
+
+
+This document assumes you have deployed Quay Enterprise with a self-signed certificate.
+
+[Appr](https://github.com/app-registry/appr) makes use of the python requests library, which by default only trusts a standard set of certificates. This prevents helm from interacting with a Quay Enterprise instance that is using a self-signed certificate.
+
+The error is reported as a connection error when attempting to interact with the registry:
+
+```
+$ helm registry version https://reg.example.com
+Api-version: .. Connection error
+Client-version: 0.3.7
+```
+
+To get around this restriction add the CA for the self signed certificate to the default list of trusted Certificate Authorities. The following links highlight how this can be accomplished:
```

Comment:
To work around this restriction, add the CA for the self-signed certificate to the default list of trusted Certificate Authorities. This process is described for the following platforms: